_docs/rest_api_specifications/pubapis_swagger.md

```diff
@@ -127,7 +127,7 @@ To create a Swagger spec file:
 
 2.  Create the Swagger spec here.
 
-    You could just customize this sample YML file shown in the Swagger editor with the weather details. However, if you're new to Swagger it will take you some time to learn the spec. For the sake of convenience, just go to this  file &mdash; <a href="http://idratherassets.com/restapicourse/swagger/swagger_weather.yml">swagger_weather.yaml</a> &mdash; and copy and paste its code into the Swagger editor. The next tutorial, [Swagger spec deep dive](pubapis_swagger_spec_deep_dive.html), will get into the nuts and bolts of the Swagger spec.
+    You could just customize this sample YML file shown in the Swagger editor with the weather details. However, if you're new to Swagger it will take you some time to learn the spec. For the sake of convenience, just go to this  file &mdash; <a href="http://idratherassets.com/restapicourse/swagger/swagger_weather.yml">swagger_weather.yml</a> &mdash; and copy and paste its code into the Swagger editor. The next tutorial, [Swagger spec deep dive](pubapis_swagger_spec_deep_dive.html), will get into the nuts and bolts of the Swagger spec.
```

Comment:
Changed the link text from "swagger_weather.yaml" to "swagger_weather.yml" so that it matches the name of the file being linked to.